docs/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>IDT form</title>
+</head>
+<link rel="stylesheet" href="style.css">
+<script src="script.js"></script>
+
+<body>
+    <section id="idt-form">
+        <h1>Interactive Learning Generator</h1>
+        <div>
+            <h3>A student project by <a href="https://github.com/Raunaq27" target=”_blank”>@Raunaq27</a> & <a
+                    href="https://github.com/Anmol135" target=”_blank”>@Anmol135</a> </h3>
+                    <a href="https://github.com/Interactive-Design-and-Technology/interactive-learning" target="_blank">Github link</a>
+            <p>This tool allows you to generate HTML, CSS, & JavaScript for interactive learning questions that can be
+                inserted into learning management systems.</p>
+            <form>
+                <label for="type">Type:</label>
```

Comment:
Validation error: `The value of the for attribute of the label element must be the ID of a non-hidden form control.`

c/o https://validator.w3.org/